tests/unit/api/test_token.py

```diff
@@ -145,7 +145,7 @@ class User:
                 id=1213,
                 username="read_user",
                 password="test",
-                scopes=[Scope("read:targets")],
+                scopes="read:settings",
```

Comment:
To fix the CI error, You need to revert it. It is the Database object Mock.

Reply:
Ohh, sorry, obvious mistake...